tests/clients/test_ngsi_v2_timeseries.py

```diff
@@ -90,36 +90,27 @@ def test_meta_endpoints(self) -> None:
             self.assertIsNotNone(client.get_version())
             self.assertIsNotNone(client.get_health())
 
-    @clean_test(fiware_service=settings.FIWARE_SERVICE,
```

Comment:
why it should be removed?

Reply:
If i remember correctly, it is because of the fiware service path. There was a discrepancy between the path in the setup and the path in the settings. I was meaning to put it back.